### PR TITLE
 #166240955 Rating Article Functionality Added

### PR DESCRIPTION
**Pull request by UhiriweAudace**
#### What does this PR do?
- In this PR, I implemented the rating article functionality
#### Description of Task to be completed?
- The logged-in users will be able to submit their ratings on a specific article
- The logged-in users will be able to view the ratings of a specific article
- The logged-in users will be able to update their ratings on a specific article
#### How should this be manually tested?
- Clone the repository, then switch to this branch `ft-rating-article-166240955`
- Install all dependencies by writing the following command: 
       - `yarn install`
- Then run the application with the following command:
       - `yarn dev`
- To run the test, simply run this command: `yarn test`
#### Any background context you want to provide?
- **N/A**
#### What are the relevant pivotal tracker stories?
- [#166240955](https://www.pivotaltracker.com/n/projects/2350798/stories/166240955)
#### Screenshots (if appropriate)
- once you need to rate an article, you should first make a login
ratingArticle

#### Questions:

### CODE REVIEW

**Review by EmyRukundo — COMMENTED** [review]
@UhiriweAudace 
   I think you should remove these comments.

File: src/redux/actions/users/login.actions.js [hunk 1]
```diff
@@ -38,6 +39,12 @@ export const login = (email, password) => async (dispatch) => {
     const user = jwt(token);
     sessionStorage.setItem('token', token);
     toast.success(`Thank you ${user.firstName}, you are now logged in successfully!`);
+    // const previousLink = sessionStorage.getItem('previousLink');
```
Comment (EmyRukundo) [inline src/redux/actions/users/login.actions.js]: these comments

**Review by EmyRukundo — APPROVED** [review]

**Comment by deschantkn** [comment]
Test comment

File: src/redux/actions/users/login.actions.js (hunk 1 above)
Comment (UhiriweAudace) [reply src/redux/actions/users/login.actions.js]: resolved

File: __tests__/redux/actions/getArticles.test.js [hunk 2]
```diff
@@ -205,15 +206,24 @@ describe('Get profile action', () => {
...
-    moxios.wait(() => {
-      const request = moxios.requests.mostRecent();
-      request.respondWith({
-        status: 200,
-        response: { profile: {} },
-      });
+    // moxios.wait(() => {
+    //   const request = moxios.requests.mostRecent();
+    //   request.respondWith({
+    //     status: 200,
```
Comment (eliemugenzi) [inline __tests__/redux/actions/getArticles.test.js]: Please delete these commented lines

File: src/assets/scss/base/_common.scss [hunk 3]
```diff
@@ -25,4 +25,13 @@
 }
 // .likeButton p{
```
Comment (eliemugenzi) [inline src/assets/scss/base/_common.scss]: Please delete these commented code

**Review by eliemugenzi — CHANGES_REQUESTED** [review]

**Review by eliemugenzi — APPROVED** [review]

**Review by rukundoeric — APPROVED** [review]

**Pull request merged**